### PR TITLE
Make some small adjustments on conversation search.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review.

### CODE REVIEW

File: shared/chat/conversation/search/index.js [hunk 1]
```diff
@@ -181,18 +181,18 @@ class ThreadSearchDesktop extends React.Component<SearchProps & Props> {
...
-                    {this.props.status === 'done' && this.props.hits.length === 0
+                    {noResults
                       ? 'No results'
                       : `${this.props.selectedIndex + 1} of ${this.props.hits.length}`}
                   </Kb.Text>
                   <Kb.Icon
-                    color={Styles.globalColors.black_50}
-                    onClick={this.props.onUp}
+                    color={noResults ? Styles.globalColors.black_35 : Styles.globalColors.black_50}
+                    onClick={!noResults ? this.props.onUp : null}
```
Comment (mmaxim) [inline shared/chat/conversation/search/index.js]: This feels a little weird to me, probably better done in the container.

Comment (adamjspooner) [reply shared/chat/conversation/search/index.js]: Good point.

Comment (adamjspooner) [reply shared/chat/conversation/search/index.js]: @mmaxim, I can't seem to get it to work. The goal is to remove the hover styling which Icon's `onClick` prop provides.

Comment (mmaxim) [reply shared/chat/conversation/search/index.js]: Ah ok, no worries then.

**Pull request merged**